### PR TITLE
#14 main page cards

### PR DESCRIPTION
**Pull request by nikitakukharchuk**
Add cards in main page.

### CODE REVIEW

**Review by visiteur85 — DISMISSED** [review]

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,5 +1,22 @@
 import * as NextImage from "next/image";
 import 'tailwindcss/tailwind.css';
+import '../src/styles/globals.css';
+import '@fontsource/roboto/300.css';
```
Comment (gitkoshelew) [inline .storybook/preview.js]: can we create one root file for fonts and import parts into it?

File: src/components/cards/Cards.tsx [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+    id: number
+    title: string
+    userName: string
+    date: string
+    status: string | null
+}
 
 export const Cards = () => {
+
+    const cards: CardsType = [
```
Comment (gitkoshelew) [inline src/components/cards/Cards.tsx]: transfer this list to mock file

File: src/components/layout/Layout.tsx [hunk 3]
```diff
@@ -13,15 +13,13 @@ export const Layout: FC<LayoutPropsType> = ({children, header = "default"}) => {
...
             display: "flex",
             flexDirection: "column",
-            fontSize: {xs: "small", sm: "medium"},
-            minHeight: "100vh",
-            bgcolor: "background.default",
+            typography: {xs: "subtitle1", sm: "fontSize"},
             gap: 3,
             p: 2
         }}>
             {header === "default"
```
Comment (gitkoshelew) [inline src/components/layout/Layout.tsx]: create map here like outside Layout
headersMap {
default: Header,
full: FullHeader
}
const HeaderComponent = headersMap[header]

and then use  <HeaderComponent />

File: src/components/cards/Card/Card.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    userName: string
+    date: string
+    status: string | null
+}
+
+export const Card: FC<PropsCardType> = ({title, date, userName, status}) => {
+
+    return (
+        <div className='border rounded-2xl shadowCard px-10 py-10 relative'>
+            <div className='border rounded-xl shadow-md absoluteCard px-5 bg-sky-400 text-white'>{status && "verifyed"}</div>
```
Comment (gitkoshelew) [inline src/components/cards/Card/Card.tsx]: looks like you shouldnt render div at all if no status

**Review by gitkoshelew — DISMISSED** [review]

**Review by visiteur85 — APPROVED** [review]

**Review by gitkoshelew — APPROVED** [review]

**Pull request merged**